miasm2/analysis/depgraph.py

```diff
@@ -246,7 +248,9 @@ def pending(self):
     def _get_modifiers_in_cache(self, depnode, force=False):
         """Recursively find nodes in the path of @depnode which are modifiers.
         Update the internal cache
-        If @depnode is already managed (ie. in @depnode_queued), abort"""
+        If @depnode is already managed (ie. in @depnode_queued), abort
+        @depnode: DependencyNode
+        @force (optionnal): boolean"""
```

Comment:
This is a protected method, so docstring can be omitted (but it's still better to add it).
But if you add it for args, please specify what are their purposes, not only their expected types.